cfme/tests/control/test_bugs.py

```diff
@@ -234,6 +234,17 @@ def vm_compliance_policy_profile(appliance, compliance_condition):
     policy.delete()
 
 
+@pytest.fixture
+def breadcrumb_locations():
```

Comment:
**Suggestion:**
I think simply `Dictionary` of `locations` would okay rather than treating it as `fixture`.

Reply:
No specific reason, I really saw them as equivalent so if you'd prefer I can just make it a constant `dict`

Reply:
@sbulage done